### DISCUSSION TITLE
Tamron 28-75mm Di III VXD G2?

### DISCUSSION THREAD

**Discussion by hubbsterx** [lensfun/lensfun#2042] [Ideas]
Hey there!
Were it possible to add a calibration file for Tamron 28-75mm Di III VXD G2?
It would be great!!
Thanks a lot!
Cheers
hubb

**Comment by tuxfanx** [comment]
The quickest way (first option) is to calibrate the lens yourself and provide us with the data via PR and upload some test images. For the calibration you need the program [Hugin](https://hugin.sourceforge.io/) and the tool [calibrate.py](https://wilson.bronger.org/lens_calibration_tutorial/) or[ lens_calibrate.py](https://pixls.us/articles/create-lens-calibration-data-for-lensfun/)
The 2nd option is to upload the corresponding images for the calibration and to wait for someone to carry out the calibration.

https://pixls.us/articles/create-lens-calibration-data-for-lensfun/
https://wilson.bronger.org/lens_calibration_tutorial/
https://wilson.bronger.org/calibration
https://wilson.bronger.org/lensfun/el_lens.html
https://www.youtube.com/watch?v=ci0BY9Bljzk

**Reply by hubbsterx** [reply]
thanks, I'll consider to upload some images
